pytest.ini

```diff
@@ -45,3 +45,7 @@ filterwarnings=
 	# https://github.com/pytest-dev/pytest/discussions/9296
 	ignore:Distutils was imported before setuptools
 	ignore:Setuptools is replacing distutils
+
+	# Workaround for pypa/setuptools#3079
```

Comment:
Here, I like to reference the upstream issue, such that when that upstream issue is fixed, it will indicate that this workaround can be removed.